values.yaml

```diff
@@ -124,10 +124,10 @@ jenkins:
           Maven:
             Image: $INTERNAL_DOCKER_REGISTRY/nuxeo/builder-java8:latest
             Privileged: true
-            RequestCpu: "1"
-            RequestMemory: "512Mi"
-            LimitCpu: "2"
-            LimitMemory: "1Gi"
+            RequestCpu: "3"
+            RequestMemory: "4Gi"
+            LimitCpu: "4"
+            LimitMemory: "4Gi"
```

Comment:
Don't you think the same value as for Nuxeo-Package-11 would be enough?
https://github.com/nuxeo/jx-platform-env/blob/master/values.yaml#L239

Or, if the build really needs 4 CPU units and 4 GB of RAM, maybe divide the request values by 2, to avoid reserving too much resources:
```
            RequestCpu: "2"
            RequestMemory: "2Gi"
            LimitCpu: "4"
            LimitMemory: "4Gi"
```

Reply:
Sorry, I took the same values than platform-11 instead of package-11 by mistake: I'll redo the commit

Reply:
Done.

And actually, I think the LimitMemory to 3G was only needed for functional tests : so if there's no FF anyway maybe I can put 2Gi instead of 3? Tell me if you'rather have this.

Reply:
Yes, the lower is the best, if it works...

Reply:
Ok, will lower for a first try